config/application.rb

```diff
@@ -26,10 +26,7 @@ class Application < Rails::Application
     # Gzip files when possible
     config.middleware.use Rack::Deflater
 
-    #HCaseflow application urls
-    config.app_urls = {"caseflow" => "https://caseflow.ds.va.gov", "efolder" => "https://efolder.cf.ds.va.gov"}
-
-    #Caseflow application official names
-    config.app_names = {"caseflow" => "Caseflow Certification", "efolder"=> "eFolder Express"}
+    #Caseflow application official names based on domain name
+    config.app_names = {"www.va.gov" => "Caseflow Certification", "efolder.cf.ds.va.gov"=> "eFolder Express"}
```

Comment:
hmm, this doesn't look quite right. we should have the certification url display `Caseflow Certification`. i was just suggesting that if we don't pass in a url, it should redirect to `www.va.gov` as a basic default.

also, can we change the name of the param to `redirect`? it will often be a long url, and not just the application url.